frontend/src/store/project-context.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
nit: good to remove unused code or add a comment declaring it for future reference

e.g.

```
// ProjectContext is not currently being used, but is kept here for future reference incase it is useful.
```